cheroot/test/test_core.py

```diff
@@ -306,14 +307,14 @@ def test_large_request(test_client_with_defaults):
         ),
         (
             b'GET / HTTPS/1.1',  # invalid proto
-            HTTP_BAD_REQUEST, b'Malformed Request-Line: bad protocol',
+            HTTP_BAD_REQUEST, b'Malformed Request-Line',
         ),
         (
             b'GET / HTTP/1',  # invalid version
-            HTTP_BAD_REQUEST, b'Malformed Request-Line: bad version',
+            HTTP_BAD_REQUEST, b'Malformed Request-Line',
         ),
         (
-            b'GET / HTTP/2.15',  # invalid ver
+            b'GET / HTTP/2.1',  # invalid ver
```

Comment:
This change is controversial. Why the RFC's spec is that it has one digit after the dot, the next paragraph gives an example of the two-digit version...

Reply:
@ian-otto ^